src/rpcdataacq.cpp

```diff
@@ -0,0 +1,737 @@
+// Copyright (c) 2010 Satoshi Nakamoto
+// Copyright (c) 2009-2012 The Bitcoin developers
+// and The Gridcoin developers
+// Distributed under the MIT/X11 software license, see the accompanying
+// file COPYING or http://www.opensource.org/licenses/mit-license.php.
+
+#include "main.h"
+#include "bitcoinrpc.h"
+#include "cpid.h"
+#include "kernel.h"
+#include "init.h" // for pwalletMain
+#include "block.h"
+#include "txdb.h"
+#include "beacon.h"
+#include "util.h"
+#include "neuralnet.h"
+#include "grcrestarter.h"
+#include "backup.h"
+#include "appcache.h"
+#include "tally.h"
+#include "contract/polls.h"
+#include "contract/contract.h"
```

Comment:
The contract/ grcrestarter, backup and boost/algorithm/string includes can be dropped.